stdlib/bytes.mli

```diff
@@ -329,6 +329,30 @@ val equal: t -> t -> bool
     always-correct {!Bytes.to_string} and {!Bytes.of_string} instead.
 *)
 
+val to_hex : bytes -> bytes
+(** Return the given byte sequence as a byte sequence of US-ASCII lowercase
+    hexadecimal digits.
+
+    @since 4.11.0
+*)
+
+val from_hex : bytes -> bytes
```

Comment:
Conversion functions are more often called `of_XXX` than `from_XXX` in the stdlib (I know, `Digest.from_hex`...).

Reply:
That's a good point - I'll change that (it's OK that `Digest.from_hex` is older)